code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -454,10 +454,20 @@
 /datum/reagent/beer2/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
 	switch(current_cycle)
-		if(1 to 50)
-			M.Sleeping(4 SECONDS)
+		if(1 to 5)
+			if(prob(25))
+				M.emote("yawn")
+		if(6 to 9)
+			M.AdjustEyeBlurry(10 SECONDS)
+			if(prob(35))
+				M.emote("yawn")
+		if(10)
+			M.emote("faint")
```

Comment:
Faint already sleeps for a little bit, worth checking how long that is, though the weaken's probably good here as well.

Reply:
The faint emote sleeps for two seconds. The weaken was paired with it in ketamine's cycle code... I assume that's so you couldn't run around for a split second before it moves onto the 11th cycle and knocks you out again?

Reply:
Yeah, I think that makes sense, especially if someone shakes you awake during it. Just wanted to make sure it was considered